modules/core/src/bitgo.ts

```diff
@@ -1261,6 +1263,11 @@ export class BitGo {
    * Verify the HMAC for an HTTP response
    */
   verifyResponse({ url: urlPath, statusCode, text, timestamp, token, hmac }: VerifyResponseOptions) {
+    if (this._skipHMACVerification) {
```

Comment:
@tylerlevine not sure if we want to also limit this to only happen in non-production environments. Thoughts?